### PR TITLE
Add Loopring deposits

### PR DESCRIPTION
**Pull request by rodyce**

### CODE REVIEW

File: src/loopring/LoopringLayer2Provider.ts [hunk 1]
```diff
@@ -98,33 +122,72 @@ class LoopringLayer2Provider implements Layer2Provider {
...
 
-  LOOPRING_REST_HOSTS_BY_NETWORK = {
-    'localhost': undefined,
-    'rinkeby': undefined,
-    'ropsten': undefined,
-    'mainnet': 'https://api3.loopring.io',
-    'goerli': 'https://uat3.loopring.io',
-    'homestead': undefined
+  getLoopringExchangeContractAddressByNetwork(network: Network): string {
+    const ret = this.LOOPRING_INFO_BY_NETWORK[network][1];
```
Comment (dougstorm) [inline src/loopring/LoopringLayer2Provider.ts:147]: maybe using lodash get method you can do this without running into undefined

https://lodash.com/docs/4.17.15#get

File: src/loopring/LoopringLayer2Wallet.ts [hunk 2]
```diff
@@ -13,61 +18,133 @@ import {
...
+    const tokenData = this.tokenDataBySymbol[deposit.tokenSymbol];
+
+    // Deposit operation parameters.
+    const from = this.address;
+    const to = this.address;
+    const tokenAddress = tokenData.address;
+    const amountInWei = ethers.utils.parseEther(deposit.amount);
+    const auxiliaryData = 0x00;
+    const overrides = {
+      gasLimit: 300_000,
```
Comment (dougstorm) [inline src/loopring/LoopringLayer2Wallet.ts:119]: Can be hardcoded in a Constant

**Comment by dougstorm** [comment]
overall looks good, left two comments

File: src/loopring/LoopringLayer2Wallet.ts (hunk 2 above)
Comment (rodyce) [reply src/loopring/LoopringLayer2Wallet.ts]: Done

**Review by dougstorm — APPROVED** [review]
LGTM

**Pull request merged**